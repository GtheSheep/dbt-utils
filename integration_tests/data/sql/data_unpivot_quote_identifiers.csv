customer_id,created_at,status,2022-01-01
123,"2017-01-01","active",1
234,"2017-02-01","active",2
567,"2017-03-01","churned",3
