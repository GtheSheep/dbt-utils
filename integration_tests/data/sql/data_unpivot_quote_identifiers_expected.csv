customer_id,created_at,prop,val
123,"2017-01-01","2022-01-01",1
123,"2017-01-01","status","active"
234,"2017-02-01","2022-01-01",2
234,"2017-02-01","status","active"
567,"2017-03-01","2022-01-01",3
567,"2017-03-01","status","churned"
